apis/apikeys/v1alpha1/apikey_type.go

```diff
@@ -41,7 +41,7 @@ var (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:categories=gcp,shortName=gcpapikeyskey;gcpapikeyskeys
 // +kubebuilder:subresource:status
-// +kubebuilder:metadata:labels="cnrm.cloud.google.com/tf2crd=true";"cnrm.cloud.google.com/managed-by-kcc=true";"cnrm.cloud.google.com/stability-level=alpha";"cnrm.cloud.google.com/system=true"
```

Comment:
With this change, can we still have resources in transition? For instance, for ApiKey AFAICT, we didn't actually make the switch to a direct CRD but I could be wrong. 

What I like about the current state of the world is that we can turn on TF/ DCL based resources to use the DirectController via the feature flag (`KCC_USE_DIRECT_RECONCILERS`) and have developers iterate between the behavior of a TF/ DCL based controller and a direct one. Can we preserve that somehow?